client.py

```diff
@@ -5,38 +5,23 @@
 client = socket.socket(
     socket.AF_INET, socket.SOCK_STREAM, socket.IPPROTO_IP
     )
+client.connect(ADDR)
 
 
-def response_error():
+def send_request(msg):
     try:
-        client.connect(ADDR)
-    except Exception:
-        return sys.stdout.write(
-            "HTTP/1.1 500 not OK\n"
-            "Content-Type: text/html; charset=utf-8\n"
-            "<!DOCTYPE html>\n"
-            "<html>\n"
-            "\t<head>\n"
-            "\t\t<title>Status 500 not OK</title>\n"
-            "\t</head>\n"
-            "\t<body>\n"
-            "\t\t<p>HTTP:500 Internal Server Error</p>\n"
-            "\t</body>\n"
-            "</html>\n"
-            )
+        client.sendall(msg)
```

Comment:
it's generally not good practice to write methods that rely on items in module scope being defined.  The fact that you use `client` here without building it explicitly in this function, or passing it in, is not a good idea.